test/navigation/OfficeSuppliesNavigatorSpec.scala

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package navigation
+
+import controllers.journeys.expenses.officeSupplies.routes.{OfficeSuppliesCYAController, OfficeSuppliesDisallowableAmountController}
+import controllers.standard.routes.JourneyRecoveryController
+import models.database.UserAnswers
+import models.{CheckMode, NormalMode}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import pages.expenses.WorkFromHomePage
+import pages.expenses.officeSupplies.{OfficeSuppliesAmountPage, OfficeSuppliesDisallowableAmountPage}
+
+class OfficeSuppliesNavigatorSpec extends AnyWordSpec with Matchers {
+
+  private val navigator = new OfficeSuppliesNavigator
+
+  private val taxYear     = 2024
+  private val userAnswers = UserAnswers("some_id")
+  private val businessId  = "some_businessId"
+
```

Comment:
taxYear, userAnswersId, emptyUserAnswers and businessId exist in the SpecBase if you want to extend that. CheckMode and NormalMode can be used without the vals

Reply:
The reason I didn't as I find AnyWordSpec more readable, but SpecBase uses AnyFreeSpec. I can use SpecBase to be consistent with the other files.